client/src/helpers/translation.js

```diff
@@ -19,3 +19,20 @@ export const handleTranslation = (
     }
   }
 }
+
+export const getNavigatorLanguage = () => {
+  let language = navigator.language
```

Comment:
Au cas où le navigateur ne supporte pas cette propriété
```suggestion
  const language = navigator?.language ?? 'en'
```